api/v1alpha1/dataset_types.go

```diff
@@ -346,13 +348,28 @@ func (dataset *Dataset) GetDataOperationInProgress(operationType string) string
 // SetDataOperationInProgress set the data operation running on this dataset,
 func (dataset *Dataset) SetDataOperationInProgress(operationType string, name string) {
 	if dataset.Status.OperationRef == nil {
-		dataset.Status.OperationRef = map[string]string{}
+		dataset.Status.OperationRef = map[string]string{
+			operationType: name,
+		}
+		return
 	}
 
-	dataset.Status.OperationRef[operationType] = name
+	dataset.Status.OperationRef[operationType] = dataset.Status.OperationRef[operationType] + "," + name
```

Comment:
need check `dataset.Status.OperationRef[operationType]` is `nil`, otherwise will get `,a`.

Reply:
It has been checked before.

Reply:
Previous just check the `OperationRef` nil, not check `OperationRef[operationType]` nil.
Consider the following scenarios:
1) dataload A initialize the `OperationRef` from nil to `{"dataload": "A"}`;
2) datamigrate B the will set the  `OperationRef` to `{"dataload": "A", "datamigrate": ",B"}`;

Reply:
Thanks for your kindly remind! it will be fixed in https://github.com/fluid-cloudnative/fluid/pull/3257